[docs]: mkdocs github action currenly broken

### Description of the documentation issue

For example this one https://github.com/kapicorp/kapitan/actions/runs/25918876725/job/76182382324

```
Run uv run mkdocs build
INFO    -  Cleaning site directory
INFO    -  Building documentation to directory: /home/runner/work/kapitan/kapitan/site
INFO    -  The following pages exist in the docs directory, but are not included in the "nav" configuration:
  - tags.md
WARNING -  Doc file 'pages/input_types/jinja.md' contains a link '../argocd/docs/README.md', but the target 'pages/argocd/docs/README.md' is not found among documentation files.
WARNING -  Doc file 'pages/input_types/jinja.md' contains a link '../dev-sockshop/docs/README.md', but the target 'pages/dev-sockshop/docs/README.md' is not found among documentation files.
WARNING -  Doc file 'pages/input_types/jinja.md' contains a link '../echo-server/docs/README.md', but the target 'pages/echo-server/docs/README.md' is not found among documentation files.
WARNING -  Doc file 'pages/input_types/jinja.md' contains a link '../examples/docs/README.md', but the target 'pages/examples/docs/README.md' is not found among documentation files.
WARNING -  Doc file 'pages/input_types/jinja.md' contains a link '../gke-pvm-killer/docs/README.md', but the target 'pages/gke-pvm-killer/docs/README.md' is not found among documentation files.
WARNING -  Doc file 'pages/input_types/jinja.md' contains a link '../global/docs/README.md', but the target 'pages/global/docs/README.md' is not found among documentation files.
WARNING -  Doc file 'pages/input_types/jinja.md' contains a link '../guestbook-argocd/docs/README.md', but the target 'pages/guestbook-argocd/docs/README.md' is not found among documentation files.
WARNING -  Doc file 'pages/input_types/jinja.md' contains a link '../kapicorp-demo-march/docs/README.md', but the target 'pages/kapicorp-demo-march/docs/README.md' is not found among documentation files.
WARNING -  Doc file 'pages/input_types/jinja.md' contains a link '../kapicorp-project-123/docs/README.md', but the target 'pages/kapicorp-project-123/docs/README.md' is not found among documentation files.
WARNING -  Doc file 'pages/input_types/jinja.md' contains a link '../kapicorp-terraform-admin/docs/README.md', but the target 'pages/kapicorp-terraform-admin/docs/README.md' is not found among documentation files.
WARNING -  Doc file 'pages/input_types/jinja.md' contains a link '../mysql/docs/README.md', but the target 'pages/mysql/docs/README.md' is not found among documentation files.
WARNING -  Doc file 'pages/input_types/jinja.md' contains a link '../postgres-proxy/docs/README.md', but the target 'pages/postgres-proxy/docs/README.md' is not found among documentation files.
WARNING -  Doc file 'pages/input_types/jinja.md' contains a link '../pritunl/docs/README.md', but the target 'pages/pritunl/docs/README.md' is not found among documentation files.
WARNING -  Doc file 'pages/input_types/jinja.md' contains a link '../prod-sockshop/docs/README.md', but the target 'pages/prod-sockshop/docs/README.md' is not found among documentation files.
WARNING -  Doc file 'pages/input_types/jinja.md' contains a link '../sock-shop/docs/README.md', but the target 'pages/sock-shop/docs/README.md' is not found among documentation files.
WARNING -  Doc file 'pages/input_types/jinja.md' contains a link '../tesoro/docs/README.md', but the target 'pages/tesoro/docs/README.md' is not found among documentation files.
WARNING -  Doc file 'pages/input_types/jinja.md' contains a link '../tutorial/docs/README.md', but the target 'pages/tutorial/docs/README.md' is not found among documentation files.
WARNING -  Doc file 'pages/input_types/jinja.md' contains a link '../vault/docs/README.md', but the target 'pages/vault/docs/README.md' is not found among documentation files.

Aborted with 18 warnings in strict mode!
Error: Process completed with exit code 1.
```

### Additional context

_No response_

--- a/docs/pages/input_types/jinja.md
+++ b/docs/pages/input_types/jinja.md
@@ -89,24 +89,24 @@ Templates will be provided at runtime with 3 variables:
         ```markdown
         | *Target*                                                               |
         |------------------------------------------------------------------------|
-        | [argocd](../argocd/docs/README.md)                                     |
-        | [dev-sockshop](../dev-sockshop/docs/README.md)                         |
-        | [echo-server](../echo-server/docs/README.md)                           |
-        | [examples](../examples/docs/README.md)                                 |
-        | [gke-pvm-killer](../gke-pvm-killer/docs/README.md)                     |
-        | [global](../global/docs/README.md)                                     |
-        | [guestbook-argocd](../guestbook-argocd/docs/README.md)                 |
-        | [kapicorp-demo-march](../kapicorp-demo-march/docs/README.md)           |
-        | [kapicorp-project-123](../kapicorp-project-123/docs/README.md)         |
-        | [kapicorp-terraform-admin](../kapicorp-terraform-admin/docs/README.md) |
-        | [mysql](../mysql/docs/README.md)                                       |
-        | [postgres-proxy](../postgres-proxy/docs/README.md)                     |
-        | [pritunl](../pritunl/docs/README.md)                                   |
-        | [prod-sockshop](../prod-sockshop/docs/README.md)                       |
-        | [sock-shop](../sock-shop/docs/README.md)                               |
-        | [tesoro](../tesoro/docs/README.md)                                     |
-        | [tutorial](../tutorial/docs/README.md)                                 |
-        | [vault](../vault/docs/README.md)                                       |
+        | \[argocd\]\(../argocd/docs/README.md\)                                     |
+        | \[dev-sockshop\]\(../dev-sockshop/docs/README.md\)                         |
+        | \[echo-server\]\(../echo-server/docs/README.md\)                           |
+        | \[examples\]\(../examples/docs/README.md\)                                 |
+        | \[gke-pvm-killer\]\(../gke-pvm-killer/docs/README.md\)                     |
+        | \[global\]\(../global/docs/README.md\)                                     |
+        | \[guestbook-argocd\]\(../guestbook-argocd/docs/README.md\)                 |
+        | \[kapicorp-demo-march\]\(../kapicorp-demo-march/docs/README.md\)           |
+        | \[kapicorp-project-123\]\(../kapicorp-project-123/docs/README.md\)         |
+        | \[kapicorp-terraform-admin\]\(../kapicorp-terraform-admin/docs/README.md\) |
+        | \[mysql\]\(../mysql/docs/README.md\)                                       |
+        | \[postgres-proxy\]\(../postgres-proxy/docs/README.md\)                     |
+        | \[pritunl\]\(../pritunl/docs/README.md\)                                   |
+        | \[prod-sockshop\]\(../prod-sockshop/docs/README.md\)                       |
+        | \[sock-shop\]\(../sock-shop/docs/README.md\)                               |
+        | \[tesoro\]\(../tesoro/docs/README.md\)                                     |
+        | \[tutorial\]\(../tutorial/docs/README.md\)                                 |
+        | \[vault\]\(../vault/docs/README.md\)                                       |
 
         ```
 

